c/include/ml-api-common.h

```diff
@@ -73,6 +73,7 @@ typedef enum {
   ML_NNFW_TYPE_TVM = 16,              /**< Apache TVM (Since 7.0) */
   ML_NNFW_TYPE_ONNX_RUNTIME = 17,     /**< Open Neural Network Exchange (ONNX) Runtime (Since 9.0) */
   ML_NNFW_TYPE_NCNN = 18,             /**< Tencent ncnn (Since 9.0) */
+  ML_NNFW_TYPE_TENSORRT = 19,         /**< NVidia Tensor-RT (Since 9.0) */
```

Comment:
Since tensorRT is a proper noun, it seems better than ML_NNFW_TYPE_TENSOR_RT. 👏

Reply:
agreed

Reply:
I prefer `ML_NNFW_TYPE_TENSORRT` and please update the comment with official name.
```suggestion
  ML_NNFW_TYPE_TENSORRT = 19,         /**< NVidia TensorRT (Since 9.0) */
```